config/puma.rb

```diff
@@ -32,14 +32,14 @@
 # Workers do not work on JRuby or Windows (both of which do not support
 # processes).
 #
-# workers ENV.fetch("WEB_CONCURRENCY") { 2 }
+workers ENV.fetch("WEB_CONCURRENCY") { 4 }
```

Comment:
- [ ] Please make sure to comment this line of code in this file, Because of this your project will not work on some Windows devices it will give the below error. Your project should be able to support all the devices.

![image](https://github.com/Dorcas126/Budget-app/assets/83803414/f04a0743-8de1-447b-a260-49e09c80124e)